esp32-hal/src/lib.rs

```diff
@@ -3,7 +3,7 @@
 pub use embedded_hal as ehal;
 pub use esp_hal_common::{pac, prelude, Delay, RtcCntl, Serial, Timer};
```

Comment:
Let's re-export `I2C` here as well unless there's a reason not to (applies to all chips).